modules/databend/databend.go

```diff
@@ -0,0 +1,141 @@
+package databend
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	databendUser        = "databend"
+	defaultUser         = "databend"
+	defaultPassword     = "databend"
+	defaultDatabaseName = "default"
+)
+
+// DatabendContainer represents the Databend container type used in the module
+type DatabendContainer struct {
+	testcontainers.Container
+	username string
+	password string
+	database string
+}
+
+func WithDefaultCredentials() testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		username := req.Env["QUERY_DEFAULT_USER"]
```

Comment:
I think you want to transfer those values to the proper env var, right? Otherwise, those vars won't take effect

Reply:
Yes, I want to make sure the ENV exists. I will fix it.